love/src/redux/reducers/summaryData.js

```diff
@@ -28,6 +32,7 @@ export default function (state = initialState, action) {
               messages: action.messages,
             },
           ],
+          withWarning: { ...state.withWarning, [`${action.csc}:${action.salindex}`]: withWarning },
```

Comment:
I think I don't quite understand what this line does.

Reply:
Here is what that line does:
1. It sets the state variable `withWarning` using the spread operator, thus it brings all the current content of the variable (`state.withWarning`) and then adds a new element to the dictionary
2. The new element has a **key** named `${action.csc}:${action.salindex}` (eg. "ATDome:0) and a **value** with the new `withWarning` obtained at line 22, which could be `true` or `false`

Reply:
Ah, ok! I see. Thanks Sebastian!